generators/root/templates/deployment/ansible/.circleci/config.yml

```diff
@@ -0,0 +1,56 @@
+version: '2.1'
+orbs:
+  terraform: circleci/terraform@3.0.1
+
+jobs:
+  # TODO: for the deploy job to work you have to:
+  #   - Add the project vault pass to the env variables of your project in CircleCI
+  #   - Generate a SSH key pair, add the private key to your project in CircleCI
+  #     and the public key to the authorized keys on the server
+  deploy:
+    docker:
+      - image: python:3.8.0
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - terraform/install:
+          terraform_version: 1.1.7
+      - run:
+          name: Setup terraform
+          command: |
+            echo "scw_access_key = $SCW_ACCESS_KEY" >> terraform.tfvars
+            echo "scw_secret_key = $SCW_SECRET_KEY" >> terraform.tfvars
+
+            terraform init --backend-config="access_key=$SCW_ACCESS_KEY" --backend-config="secret_key=$SCW_SECRET_KEY"
+          working_directory: ~/project/terraform/staging
+      - run:
+          name: Setup ansible
+          command: |
+            apt-get update
+            apt-get install --yes jq
+
+            pip install ansible==2.9.6 jmespath
+
+            echo "$VAULT_PASS" > vault_pass.txt
+
+            git submodule update --init
+
+            ssh-keyscan -H $(cd ../terraform/staging/ && terraform output --json hosts | jq --raw-output .server) >> ~/.ssh/known_hosts
+          working_directory: ~/project/ansible
+      - run:
+          command: ansible-playbook --inventory staging.py deployment_ci.yaml
```

Comment:
Shouldn't we run a terraform apply command before this one ?

Reply:
Well, I decided to only add the deployment to the CI and not the provisioning since running the ansible provisioning takes time that is not needed most of the time.

I think it would be strange to run the Terraform, not run the ansible provisioning and then run the the ansible deployment.

We could probably work on something to only run the provisioning when changed but it would require a lot more work than what is done on this PR.